org.eclipse.scout.rt.platform.test/src/main/java/org/eclipse/scout/rt/testing/platform/TestingDefaultPlatform.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2010-2022 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+package org.eclipse.scout.rt.testing.platform;
+
+import java.util.Set;
+
+import org.eclipse.scout.rt.platform.DefaultPlatform;
+import org.eclipse.scout.rt.platform.internal.BeanFilter;
+import org.eclipse.scout.rt.platform.internal.BeanManagerImplementor;
+import org.eclipse.scout.rt.platform.inventory.ClassInventory;
+import org.eclipse.scout.rt.platform.logger.LoggerShutdownPlatformListener;
+import org.eclipse.scout.rt.testing.platform.runner.RunWithNewPlatform;
+
+/**
+ * Replacement for {@link DefaultPlatform} which is used for {@link RunWithNewPlatform} if no other platform is
+ * specified.
+ */
+public class TestingDefaultPlatform extends DefaultPlatform {
```

Comment:
Maybe extend PlatformImplementor directly, as DefaultPlatform is an empty class.

Reply:
Is this a must-have? Wouldn't it be better if we keep the extension to `DefaultPlatform` if maybe at some point in time methods are overwritten/added there? Did not change this code yet.

Reply:
Not a must-have, just a suggestion. Your argument is valid, so keep it extending DefaultPlatform.